frontend/src/views/CreateQuestionView.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <div class="container">
+    <div class="ma-2 ml-0"><h1>新しい質問をする</h1></div>
+    <v-card>
+      <v-form>
+        <v-text-field
+          label="タイトル"
+          placeholder="質問のタイトルを入力"
+          v-model="title"
+          required
+          class="ma-3"
+        ></v-text-field>
+        <div class="mx-3"><MdEditor v-model="newQuestionContent" :language="language" /></div>
+        <v-autocomplete
+          label="質問のタグを選択"
+          :items="tags"
+          item-title="name"
+          item-value="id"
+          class="mx-3 mt-6"
+          v-model="selectedTags"
+          clearable
+          multiple
```

Comment:
入力文字列に合致するタグが見つからなかった時に表示される文章がデフォルトのままになっていて少しわかりにくいかも

```suggestion
          multiple
          no-data-text="タグが見つかりません"
```

とかするとよさそう